src/components/Data/Indexes/ModalCreate.vue

```diff
@@ -0,0 +1,111 @@
+<template>
```

Comment:
If the logic with props is to complex, if this modal is too different from the original Modal: I'm agree to keep two differents modals